src/main/java/hudson/plugins/sshslaves/SSHLauncher.java

```diff
@@ -1223,6 +1224,7 @@ public String getWorkDirParam(@NonNull String workingDirectory){
     }
 
     @Extension
+    @Symbol({"sSHLauncher", "ssh"})
```

Comment:
With such ordering `sSHLauncher` will be a default alias for export. I suggest keeping it only for compatibility purposes

```suggestion
    @Symbol({"ssh", "sSHLauncher"})
```